.github/ISSUE_TEMPLATE/new_source_plugin.yml

```diff
@@ -1,13 +1,13 @@
 name: 🔄 New Source Plugin
```

Comment:
This file should also be renamed for consistency

Reply:
Done!